File: src/avaliacaoPPGI/PublicacaoConferencia.java
package avaliacaoPPGI;

import java.util.ArrayList;

public class PublicacaoConferencia extends Publicacao {

	private int numero;
	private String localConferencia;
	
	public PublicacaoConferencia(int ano, Veiculo veiculo, String titulo, ArrayList<Docente> autores, 
			int numero, String localConferencia, int paginaInicial, int paginaFinal) {
		super(ano, veiculo, titulo, autores, paginaInicial, paginaFinal);
		this.numero = numero;
		this.localConferencia = localConferencia;
	}
	
	public int getNumero() {
		return numero;
	}
	public void setNumero(int numero) {
		this.numero = numero;
	}
	
	public String getLocalConferencia() {
		return localConferencia;
	}
	
	public void setLocalConferencia(String localConferencia) {
		this.localConferencia = localConferencia;
	}
	
	@Override
	public String toString() {
		return "PublicacaoConferencia [veiculo=" + veiculo + "numero=" + numero + ", localConferencia=" + localConferencia + "]";
	}
	
}

File: julia/cod/PublicacaoConferencia.jl
using Base
include("Publicacao.jl")

struct PublicacaoConferencia <: Publicacao
    @atributos_Publicacao
    numero::Int64
    local_ocor::String

    PublicacaoConferencia(ano::Int, veiculo::Veiculo, titulo::String,
    autores::Array{Docente}, pg_inicial::Int64, pg_final::Int64,
    numero::Int64, local_ocor::String) = new(Date(ano), veiculo, titulo,
    autores, pg_inicial, pg_final, numero, local_ocor)
end

numero(p::PublicacaoConferencia) = p.numero
local_ocor(p::PublicacaoConferencia) = p.local_ocor